paper/impact-of-instability.tex

```diff
@@ -61,7 +61,7 @@
 \JournalInfo{$ $ } % Journal information
 \Archive{Pre-print} % Additional notes (e.g. copyright, DOI, review/research article)
 
-\PaperTitle{Numerical Instabilities in Analytical Pipelines Compromise the Reliability of Network Neuroscience} % Article title
+\PaperTitle{Numerical Instabilities Lead to Meaningful Variability in Brain Networks} % Article title
```

Comment:
How about 'Large and Meaningful', to suggest both the positive and negative aspects of variability?

Reply:
I would vote for 'Numerical Instabilities in Analytical Pipelines Lead to Large and Meaningful Variability in Brain Networks'

Reply:
👍